vmdb/lib/miq_automation_engine/service_models/miq_ae_service_miq_request.rb

```diff
@@ -28,5 +28,8 @@ def set_message(value)
       object_send(:update_attribute, :message, value)
     end
 
+    def description=(new_description)
+      object_send(:update_attribute, :description, new_description)
```

Comment:
@Fryguy Automate is normally used by admins scripting things and not so much developers so the idea has been to keep things simple for them where possible.  We do not expose the `.save` method on the service objects which are largely read-only properties/relationships except where we enable them to update fields like this.

@tinaafitz This should be: `object_send(:update_attributes, :description => new_description)`  Would be nice to fix the set_message method as well.